webpack.config.js

```diff
@@ -22,7 +22,7 @@ module.exports = (env, argv) => {
   ]
 
   return {
-    watch: true,
+    watch: false,
```

Comment:
Why turn off the test watch here?

Reply:
It won't finish building if set to watch.